docs/datasets/icelandic.md

```diff
@@ -629,6 +629,60 @@ You can evaluate this dataset directly as follows:
 $ scandeval --model <model-id> --dataset mmlu-is
 ```
 
+### Unofficial: IcelandicKnowledge
+This dataset is based on the IcelandicQA dataset, which was published [here](https://huggingface.co/,datasets/mideind/icelandic_qa_scandeval), but is here phrased as a knowledge dataset. The candidate answers has been generated by GPT-4o.
+
+The original dataset has 2,000 samples, but only 1,997 unique questions, and the total length of this dataset is therefore 1,997. The split is given by 845 / 128 / 1024 for train, val, and test, respectively.
+
+Here are a few examples from the training split:
+
+```json
+{
+    "text": "Hvaða gamla verðeining var jafngildi einnar kýr að verðmæti?\nSvarmöguleikar:\na. Sauðfé\nb. Kúgildi\nc. Mjólkurtollur\nd. Hrossgildi",
+    "label": "b"
+}
+```
+```json
+{
+    "text": "Hvenær komu Íslendingar fyrst til Gimli í Manitoba?\nSvarmöguleikar:\na. 15. september 1875\nb. 25. október 1874\nc. 10. október 1876\nd. 21. október 1875",
+    "label": "d"
+}
+```
+```json
+{
+    "text": "Hvaða ár var byggingin sem gaf Barónsstíg í Reykjavík nafn reist?\nSvarmöguleikar:\na. 1901\nb. 1897\nc. 1899\nd. 1898",
+    "label": "c"
+}
+```
+
+When evaluating generative models, we use the following setup (see the
+[methodology](/methodology) for more information on how these are used):
+
+- Number of few-shot examples: 4
+- Prefix prompt:
+  ```
+  Eftirfarandi eru textar með tilheyrandi spurningum og svörum.
+  ```
+- Base prompt template:
+  ```
+  Texti: {text}
+  Spurning: {question}
+  Svaraðu með að hámarki 3 orðum: {label}
+  ```
+- Instruction-tuned prompt template:
+  ```
+  Texti: {text}
+
+  Svaraðu eftirfarandi spurningu um textann að hámarki í 3 orðum.
+
+  Spurning: {question}
+  ```
```

Comment:
These prompts don't match the multiple choice format, but are instead the reading comprehension prompts.